ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/CloudSourceIntegrations/OcmIntegrationForm.tsx

```diff
@@ -0,0 +1,419 @@
+                                                rel="noreferrer"
+                                            >
+                                                Configuring access to clusters in OpenShift Cluster
+                                                Manager
+                                            </Button>{' '}
+                                            for more information.
+                                        </FlexItem>
+                                        <FlexItem>
+                                            <em>
+                                                Service accounts are the preferred authentication
+                                                method over the deprecated API token.
+                                            </em>
+                                        </FlexItem>
+                                    </Flex>
+                                }
+                                ariaLabel="Help for client secret"
+                            />
+                        }
+                        fieldId="cloudSource.credentials.clientSecret"
+                        touched={touched}
+                        errors={errors}
+                    >
+                        <TextInput
+                            isRequired={
+                                values.updateCredentials && !values.cloudSource.credentials.secret
+                            }
+                            type="password"
+                            id={`cloudSource.credentials.clientSecret`}
+                            value={values.cloudSource.credentials.clientSecret}
+                            onChange={(event, value) => onChange(value, event)}
+                            onBlur={handleBlur}
+                            isDisabled={!isEditable || !values.updateCredentials}
+                            placeholder={
+                                values.updateCredentials
+                                    ? ''
+                                    : 'Currently-stored client secret will be used.'
+                            }
+                        />
+                    </FormLabelGroup>
+                    <FormLabelGroup
+                        isRequired={values.updateCredentials}
+                        label="API token (deprecated)"
+                        labelIcon={
+                            <IntegrationHelpIcon
+                                helpTitle="API token"
+                                helpText={
+                                    <Flex direction={{ default: 'column' }}>
+                                        <FlexItem>
+                                            <Button
+                                                variant="link"
+                                                isInline
+                                                component="a"
+                                                href="https://console.redhat.com/openshift/token"
```

Comment:
An observation, not necessarily a change request, unlike the live links for **Client ID** and **Client secret** this link redirects for me to **We need a little more information** page.

Reply:
It redirects to a console page that gives direct access to the token, but it requires a login to the Red Hat console (which it should prompt for if you're not logged in).